python/poetry/Portfile

```diff
@@ -23,9 +23,9 @@ long_description        Poetry: Dependency Management for Python. \
 
 homepage                https://python-poetry.org/
 
-checksums               rmd160  ba5ebd7411fb53f3d36b33a951a6910db3212023 \
-                        sha256  23519cc45eb3cf48e899145bc762425a141e3afd52ecc53ec443ca635122327f \
-                        size    1514978
+checksums               rmd160  fbcb2be0303defbb4e3710b64c5b811387c48c62 \
+                        sha256  49cceb3838104647c3e1021f3a4f13c6053704cc18d33f849a90fe687a29cb73 \
+                        size    1516966
 
 variant python38 conflicts python39 python310 python311 description {Use Python 3.8} {}
```

Comment:
some of these variants are missing the `python312` and `python311` in the `conflicts` list